### ISSUE TITLE
images are broken when rendering on the client

### ISSUE THREAD

**Issue by MasseGuillaume** [scalacenter/scaladex#452]
see https://index.scala-lang.org/wix/accord/accord-core/0.6.1?target=_2.12

**Comment by MichaelViveros** [comment]
The first image in that readme is a relative link to `assets/accord-logo-light.png` (see the [raw markdown](https://raw.githubusercontent.com/wix/accord/master/README.md)).

Other repos whose images get properly rendered in their scaladex page have absolute links. Ex. https://index.scala-lang.org/thoughtworksinc/deeplearning.scala/any/1.0.0-M0?target=_2.11, first image in readme links to http://deeplearning.thoughtworks.school/assets/images/logo-text-black.png.

So not sure if this is a bug (should Scaladex be responsible for rendering images with relative links in the readme on the project page?).

**Comment by julienrf** [comment]
Do we have an example of a README that uses relative URLs for images?

**Comment by rusucosmin** [comment]
Yes, this [one](https://raw.githubusercontent.com/wix/accord/master/README.md): `[![Accord](assets/accord-logo-light.png)](http://wix.github.io/accord)`.
It looks [rendered](https://index.scala-lang.org/wix/accord/accord-core/0.6.1?target=_2.12) correctly.

I think the relative URLs are fixed [here](https://github.com/scalacenter/scaladex/blob/master/data/src/main/scala/ch.epfl.scala.index.data/github/GithubReadme.scala#L7), and this code was introduced as part of this [related issue](https://github.com/scalacenter/scaladex/issues/522).
